alchemist/alchemist-incarnation-biochemistry/src/test/java/it/unibo/alchemist/test/TestBioRect2DEnvironmentNoOverlap.java

```diff
@@ -0,0 +1,761 @@
+package it.unibo.alchemist.test;
```

Comment:
Yes, sure. Only one question: what should I do for magic numbers? Save them in a global field or something like that?